zeus/metric.py

```diff
@@ -63,7 +85,7 @@ def __init__(
         prometheus_url: str,
         job: str,
         gpu_bucket_range: Sequence[float] = [50.0, 100.0, 200.0, 500.0, 1000.0],
-        cpu_bucket_range: Sequence[float] = [10.0, 20.0, 50.0, 100.0, 200.0],
+        cpu_bucket_range: Sequence[float] = [10.0, 50.0, 100.0, 500.0, 1000.0],
```

Comment:
You updated the default range, but you never reflected the change in any docstring or doc. (1) Update the docstring, and (2) remove the defaults listed in `measure.md` and instead point people to the API reference page for defaults.